backend/src/routes/public.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const router = express.Router()
+const { getSystemMessages } = require('../components/public')
+
+router.get('/system-messages', (req, res) => {
```

Comment:
My only concern with a public route that hits Dynamics is the potential to abuse it via DOS attacks. Now, the risk of overwhelming Dynamics is really minimal since we are caching data anyways. But I think this might be a good opportunity to add rate limiting (https://www.npmjs.com/package/express-rate-limit) support to the backend, specifically to this endpoint. Since we have a distributed backend you would also need https://www.npmjs.com/package/rate-limit-redis. We can probably add a separate NFR ticket for this for one of us to work on.

Reply:
I've added this rate limit to app.js. could you please have a look? I'm using the recommended setting in the express-rate-limit document.